src/stream/src/executor/monitor/in_mem_stats.rs

```diff
@@ -0,0 +1,86 @@
+// Copyright 2025 RisingWave Labs
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+use std::collections::HashMap;
+use std::sync::Arc;
+use std::sync::atomic::AtomicU64;
+
+use parking_lot::RwLock;
+
+pub type Count = Arc<AtomicU64>;
+
+#[derive(Clone)]
+pub struct CountMap(Arc<RwLock<HashMap<u64, Count>>>);
+
+impl CountMap {
+    pub(crate) fn new() -> Self {
+        let inner = Arc::new(RwLock::new(HashMap::new()));
+        #[cfg(all(not(test), not(madsim)))]
+        {
+            let inner = inner.clone();
+            tokio::spawn(async move {
+                loop {
+                    tokio::time::sleep(tokio::time::Duration::from_secs(60)).await;
+                    if Self::should_gc(&inner.read()) {
+                        Self::gc(&mut inner.write());
+                    }
```

Comment:
Maybe we can add some logs here to monitor the map size.

Reply:
fixed https://github.com/risingwavelabs/risingwave/commit/cc4bc13811268cf3f6e3e883dfd22bfd001dd53a